lib/oli_web/live/user_login_live.ex

```diff
@@ -69,7 +69,7 @@ defmodule OliWeb.UserLoginLive do
           <Components.Auth.login_form
             title="Sign In"
             form={@form}
-            action={~p"/users/log_in?#{[request_path: ~p"/workspaces/instructor"]}"}
+            action={~p"/users/log_in"}
```

Comment:
This was a somewhat-related issue, where users were incorrectly being sent to the instructor workspace if they were logging in with the generic sign in page. This method will instead relegate that determination to the login redirection logic instead of incorrectly specifying it here